app/frontend/src/theme.ts

```diff
@@ -57,7 +57,7 @@ const themeOptions: ThemeOptions = {
   shape: {
     borderRadius,
     navPaddingDesktop: spacing(10),
-    navPaddingMobile: spacing(8),
+    navPaddingMobile: spacing(10),
```

Comment:
This should exactly be the navbar height (3.5), some pages may not want padding between the bar and content (for example, if there is a banner image)

Reply:
Why do some pages not want padding between bar and content? (Which pages wouldn't want that?)

Reply:
Communities page wouldn't because of the banner image

Reply:
It's bad design if we need to go to every page to add spacing due to one page not wanting that spacing.

Reply:
Fair, how about separating Nav bar height and Nav bar padding variables so in those instances, we can do a negative margin accurately (or is there a better way?)

Reply:
Fair, how about separating Nav bar height and Nav bar padding variables so in those instances, we can do a negative margin accurately (or is there a better way?)